docs/Installation/get_started.md

```diff
@@ -0,0 +1,463 @@
+# Get Started
+## Prerequisite
+* Mac OS X 10.7 (Lion) or newer or a linux OS (tested on ubuntu 18.04)
+* Git installed with a github account
+* an [ssh key](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) added to [github](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account)
+    * to see if SSL key has been loaded `ssh-add -L` 
+* Docker
+    * installation [guide](https://docs.docker.com/v17.12/install/#server)
+    * verify installation with `$docker --version`
+* Python3
+    * installation [guide](https://www.python.org/downloads/)
+    * verify installation with `$python3 --version`
+* pip3  
+    * `$sudo apt install python3-pip`
+* click python package  
+    *  `$pip3 install click `  
+
+## philosophy
+JumpscaleX aka JSX was created at the beginning so that junior system administrators can easily use this tool to manage resources, provision machines, create and deploy containers.
+
+It will evolve even further to be your digital representation that will always be available. Right now this tool works on your machine but soon it will be automatically deployed on the TF grid.
+This 3bot, as we call it, will make sure your applications are properly running by creating, monitoring and maintaining your ressources at any time on your behalf.
+It will even be able to engage in trade with others and this network of 3bots will act as a decentalized exchange.
+  
+Don't worry nobody except you can take control over your 3bot as all your config files are encrypted (securely stored) and your keys never leaves your device.
+
+
+# Installation
+
+## Launch a Jumpscale X container
+
+### How it works
+The python script will load a docker image and start a docker container on your machine. You just need to answer a few questions during the install script. The container will bind a volume between the directory that holds the JSX code and the container (at /sandbox/code). You will be able to connect securely via ssh to this container and launch the shell (aka kosmos) to manipulate the Jumpscale X library .
+
+```bash
+#get the installer
+# TODO CHANGE BRANCH WITH MASTER
+curl https://raw.githubusercontent.com/threefoldtech/jumpscaleX/development_installer/install/jsx.py?$RANDOM > /tmp/jsx.py ; \
+#install
+python3 /tmp/jsx.py container-install; \
+cp /tmp/jsx.py /tmp/jsx; \
```

Comment:
Why do you download first download the script as `jsx.py` execute it with this name, then rename it to `jsx`?
This seems a bit cumbersome.
Also `chmod 777` is a bit overkill, `chmod +x` is enough.
Also if we want to have jsx as a command we can call after the install, we should also show how to move it at a location in `$PATH`